src/com/xilinx/rapidwright/util/Params.java

```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ * Copyright (c) 2023, Advanced Micro Devices, Inc.
+ * All rights reserved.
+ *
+ * Author: Chris Lavin, AMD AECG Research Labs.
+ *
+ * This file is part of RapidWright.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.xilinx.rapidwright.util;
+
+/**
+ * Aims to be a centralized helper class to manage global RapidWright settings.
+ */
+public class Params {
+
+    public static String RW_DECOMPRESS_GZIPPED_EDIF_NAME = "RW_DECOMPRESS_GZIPPED_EDIF";
```

Comment:
Does a better name for this parameter exist? I fear that this name could (at a glance) be inferred to mean that RapidWright will refuse to open any gzipped EDIFs at all unless enabled. Ideas:
```
RW_DECOMPRESS_GZIPPED_EDIF_TO_DISK
RW_GZIPPED_EDIF_USE_DISK
RW_FAST_GZIP_EDIF_PARSING
```

Reply:
Refactored to `RW_DECOMPRESS_GZIPPED_EDIF_TO_DISK`